src/webhooks/pubsub/stalebot.test.ts

```diff
@@ -79,11 +83,33 @@ But! If you comment or otherwise update it, I will reset the clock, and if you r
     ]);
   });
 
+  it('should not close stale PR that has no activity for less than a week and has `Stale` label', async function () {
+    const issueUpdateSpy = jest.spyOn(org.api.issues, 'update');
+    org.api.issues.listForRepo = () => [];
+    org.api.pulls.list = () => [{ ...issueInfo, labels: [STALE_LABEL] }];
+    await triggerStaleBot(org, moment('2023-04-10T14:28:13Z').utc());
```

Comment:
nit: maybe I'm missing it, but is there somewhere in this files that lists what the current time for the tests is?

Reply:
Yep, it's at the top as a part of the `issueInfo` variable